contents/blog/data-council-meetup/index.mdx

```diff
@@ -0,0 +1,33 @@
+---
+title: Meet Us at Data Council Austin
+date: 2023-02-24
+author: Michael Robinson
+image: ./image.svg
+banner: ./banner.svg
+description: The OpenLineage community will be gathering on March 30th at Data Council Austin -- join us!
+---
+
+[Data Council](https://www.datacouncil.ai/) will be holding its only 
+[conference of 2023](https://www.datacouncil.ai/austin) in Austin, Texas, on 
+March 28-30th, and OpenLineage will be there. 
+
+Among the speakers at this year's event will be OpenLineage Project Lead and 
+[Astronomer](https://astronomer.io) Chief Architect [Julien Le Dem](https://www.datacouncil.ai/talks/ten-years-of-building-open-source-standards?hsLang=en).
+
+Join Julien and other members of the OpenLineage community on March 30th from 
+12:00-1:30 pm at the [AT&T Hotel and Conference Center](https://meetattexas.com/hotel) 
+on the campus of UT Austin for a community meetup.
+
+Join us for a wide-ranging, open-ended discussion of the past, present and future 
+of the OpenLineage spec. Meet other members of the data ecosystem, learn more 
+about the project's goals and design, and participate in a conversation about 
+what's next.
+
+For more information about Data Council Austin, visit their 
+[website](https://www.datacouncil.ai/austin). Watch this space for additional 
+details about the meetup as the date draws near. We hope to see you there.
+
```

Comment:
I think we should have a strong call to action that uses our registration code - perhaps "if you haven't registered for this event yet, click this link and use the code OpenLineage20"